src/tilecache.js

```diff
@@ -163,15 +163,14 @@
 
         /**
          * @private
-         * Access of the data by drawers, synchronous function.
-         *
-         * When drawers access data, they can choose to access this data as internal copy
+         * Access of the data by drawers, synchronous function. Should always access a valid main cache, e.g.
+         * cache swap should be atomic.
          *
          * @param {OpenSeadragon.DrawerBase} drawer
-         *   until 'setData' is called
+         * @param {OpenSeadragon.Tile} tileDrawn
```

Comment:
The name `tileDrawn` sounds like a boolean to me... A tile that had been drawn would be `drawnTile`. Or is a tile that is going to be drawn? That might be more like `tileToDraw` or something. That said, maybe we're already using this terminology elsewhere in which case it's probably just as well to stay consistent with it. 

BTW, it might be nice to comment what this parameter is for, as well as the parameter above it.